# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: layout/build/nsLayoutCID.h
# Commit: b3f0e5c7874d
# Full Hash: b3f0e5c7874dbd470d96e476e116740e4ca35925
# Author: David Zbarsky <dzbarsky@gmail.com>
# Date: 2012-08-06 14:40:43
# Regressor Bug: 779684
# File Overlap Count: 1
# Description:
#   Bug 779684: Create a stack-based nsFrameIterator class out of nsIFrameTraversal and nsIFrameEnumerator r=roc
# ==============================================================================

diff -r 4bc7154190dd -r b3f0e5c7874d layout/build/nsLayoutCID.h
--- a/layout/build/nsLayoutCID.h	Sun Aug 05 22:55:18 2012 -0400
+++ b/layout/build/nsLayoutCID.h	Sun Aug 05 23:00:55 2012 -0400
@@ -6,10 +6,6 @@
 #ifndef nsLayoutCID_h__
 #define nsLayoutCID_h__
 
-// {1691E1F4-EE41-11d4-9885-00C04FA0CF4B}
-#define NS_FRAMETRAVERSAL_CID \
-{ 0x1691e1f4, 0xee41, 0x11d4, { 0x98, 0x85, 0x0, 0xc0, 0x4f, 0xa0, 0xcf, 0x4b } }
-
 /* a6cf90f9-15b3-11d2-932e-00805f8add32 */
 #define NS_LAYOUT_DEBUGGER_CID \
  { 0xa6cf90f9, 0x15b3, 0x11d2,{0x93, 0x2e, 0x00, 0x80, 0x5f, 0x8a, 0xdd, 0x32}}